### DISCUSSION TITLE
Does the current Email Feature support OAuth?

### DISCUSSION THREAD

**Discussion by weirdyang** [OrchardCMS/OrchardCore#15507] [Q&A]
Hi,

I have a client who needs to use modern authentication/OAuth2 for SMTP, does the current out of the box Email feature support this?

**Comment by Skrypt** [comment]
Mailkit supports it. The feature is probably not using it though.

**Comment by hishamco** [comment]
AFAIK nothing related to OAuth on the current email implementation
